python_scripts/extension_documentation.py

```diff
@@ -89,23 +103,19 @@ def print_hooks():
 
     """
     i = 1
-    with open("{}/WikispeechHooks.php".format(extension_path)) as f:
+    with open("{}/WikispeechHooks.php".format(args.extension_path)) as f:
```

Comment:
Also. Is this the only Wikispeech specific line in the code? if so maybe pass it as an argument?

I could see `parser.add_argument("--hooks", "-e", default="WikispeechHooks")` and then using the `AutoloadClasses` to get the path.

(or if ambitious look for a key ending in `Hooks` in `AutoloadClasses` and skip the argument)

Reply:
Replaced it reading hooks from extension.json. It's easier (no need to find and parse hooks file) and should be more correct since those are the actual hooks that are used.

Reply:
way saner